src/CloudService/examples/Restart-AzCloudServiceRoleInstance.md

```diff
@@ -1,7 +1,7 @@
 ### Example 1: Restart role instance of a cloud service
 
 ```powershell
-PS C:\> Restart-AzCloudService -ResourceGroupName "ContosOrg" -CloudServiceName "ContosoCS" -RoleInstanceName "ContosoFrontEnd_IN_0"
```

Comment:
`src/CloudService/docs/Restart-AzCloudServiceRoleInstance.md` should also be updated. Please update the docs in generation with autorest but not manully.